microsoft-365/enterprise/ms-cloud-germany-transition-add-pre-work.md

```diff
@@ -24,69 +24,104 @@ description: "Summary: Pre-work when moving from Microsoft Cloud Germany (Micros
+### Exchange Online Hybrid configuration
 
+**Applies to:** All customers using an active Exchange Hybrid Configuration with Exchange servers on premises<br>
+**When applied**: Any time before phase 5 starts
 
 | Step(s) | Description | Impact |
 |:-------|:-------|:-------|
-| Limit SharePoint 2013 workflows, use during the SharePoint Online migration. | Reduce SharePoint 2013 workflows and complete in-flight workflows before transitions. | Inaction may result in user confusion and help desk calls. |
+| Update to the latest version of the Hybrid Configuration Wizard (HCW) anytime before your tenant is entering migration stage 5. You may start this activity immediately after receiving the message center notification that your Office 365 tenant migration has begun (phase 1).<br>Your Exchange administrator must uninstall previous versions of the HCW, and then install and execute the latest version (17.0.5378.0 or higher) from [https://aka.ms/hybridwizard](https://aka.ms/hybridwizard). |<ul><li>The latest version of the HCW includes necessary updates to support the Exchange Online migration from the Microsoft Cloud Deutschland instance to Office 365 Global Services.</li><li> Updates include changes to on-premises certificate settings for the _Send connector_ and the _Receive connector_.</li><li>When executing the HCW before phase 5, select "Office 365 Germany" on the 2nd page of the HCW under _Office 365 Exchange Online_ in the listbox below  _My Office 365 organization is hosted by_</li><li>**NOTE**: Upon completion of your Office 365 tenant  migration after phase 9, you will remove and re-install the HCW again, this time using "Office 365 Worldwide" settings on the 2nd page of the HCW to complete your Hybrid setup with the Exchange Online global service.</li></ul>|Failure to run the HCW before Phase 5 (Exchange migration) may result in service or client failure. |
+| Establish AuthServer on-premises pointing to global Security Token Service (STS) for authentication | This ensures that authentication requests for Exchange availability requests from users in migration state that target the hybrid on-premises environment are authenticated to access the on-premises service. Similarly this will ensure authentication of requests from on-premises to Office 365Global services endpoints. | After Azure AD migration (phase 2) is complete, the administrator of the on-premises Exchange (hybrid) topology must add a new authentication service endpoint for the Office 365 Global services. With this command from Exchange PowerShell, replace `<TenantID>` with your organization's tenant ID found in Azure portal on Azure Active Directory.<br>`New-AuthServer GlobalMicrosoftSts -AuthMetadataUrl https://accounts.accesscontrol.windows.net/<TenantId>/metadata/json/1`<br> Failing to complete this task may result in hybrid free-busy requests failing to provide information for mailbox users who have been migrated from Microsoft Cloud Deutschland to Office 365 services.  |
```

Comment:
```suggestion
| Establish AuthServer on-premises pointing to global Security Token Service (STS) for authentication | This ensures that authentication requests for Exchange availability requests from users in migration state that target the hybrid on-premises environment are authenticated to access the on-premises service. Similarly, this will ensure authentication of requests from on-premises to Office 365 Global services endpoints. | After Azure AD migration (phase 2) is complete, the administrator of the on-premises Exchange (hybrid) topology must add a new authentication service endpoint for the Office 365 Global services. With this command from Exchange PowerShell, replace `<TenantID>` with your organization's tenant ID found in the Azure portal on Azure Active Directory.<br>`New-AuthServer GlobalMicrosoftSts -AuthMetadataUrl https://accounts.accesscontrol.windows.net/<TenantId>/metadata/json/1`<br> Failing to complete this task may result in hybrid free-busy requests failing to provide information for mailbox users who have been migrated from Microsoft Cloud Deutschland to Office 365 services.  |
```

Reply:
Insert a comma after "Similarly" in the fourth line.
Insert a space between "365" and "Global" in the fifth line.
Insert "the" before "Azure" in the eighth line.